docs/source/conf.py

```diff
@@ -68,9 +71,9 @@
     'logo_name': True,
     'show_powered_by': False,
     'extra_nav_links': {
-        'Report PodmanPy Issue': 'http://github.com/containers/podman-py/issues',
-        'Podman Reference': 'http://docs.podman.io',
-        'Podman on github': 'http://github.com/containers/podman',
+        'Report PodmanPy Issue': 'https://github.com/containers/podman-py/issues',
+        'Podman Reference': 'https://docs.podman.io',
+        'Podman on github': 'https://github.com/containers/podman',
```

Comment:
I just changed this recently from http to https, I hope it didn't mess you up too much.